### ISSUE TITLE
Env file in NPM package

### ISSUE THREAD

**Issue by radmen** [muxinc/mux-node-sdk#49]
Hey folks,

I've noticed that the npm package holds `.env-dylan` file with MUX tokens. It's not in the repo so I guess it got there during the packaging. Should it be there? :)

Steps to reproduce:

```
$ npm i @mux/mux-node
$ ls -lah node_modules/@mux/mux-node/
total 48K
drwxr-xr-x 3 radmen radmen 4,0K 04-30 10:53 .
drwxr-xr-x 3 radmen radmen 4,0K 04-30 10:53 ..
-rw-r--r-- 1 radmen radmen 2,1K 1985-10-26  CHANGELOG.md
-rw-r--r-- 1 radmen radmen  143 1985-10-26  .env-dylan
-rw-r--r-- 1 radmen radmen 1,2K 1985-10-26  .esdoc.json
-rw-r--r-- 1 radmen radmen 1,1K 1985-10-26  LICENSE
-rw-r--r-- 1 radmen radmen 2,4K 04-30 10:53 package.json
-rw-r--r-- 1 radmen radmen 8,8K 1985-10-26  README.md
drwxr-xr-x 6 radmen radmen 4,0K 04-30 10:53 src
-rw-r--r-- 1 radmen radmen  267 1985-10-26  .travis.yml
```

**Comment by GeneticGenesis** [comment]
Hey @radmen, thanks so much for reaching out.

Indeed that file shouldn't be there. We've deactivated the credentials contained in it, and we'll investigate how they got there.

Thanks again.

**Comment by dylanjha** [comment]
I ran into unexpected `npm publish` behavior. I just tested this out with a fresh package and I'm seeing that `npm publish` will respect a local `.gitignore` but not a `~/.gitignore_global`. That's what happened here.

We'll be moving to publishing via CI so that we're not running `npm publish` from dev machines. @radmen thank you so much now you [definitely deserve swag](https://github.com/muxinc/mux-node-sdk/pull/48#issuecomment-617036026), can I get in touch?
